### PR TITLE
add_DMM-Player-For-Chrome1.5.0.9

### PR DESCRIPTION
**Pull request by yutarody**
#### Adding a new cask

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/dmm-player-for-chrome.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'dmm-player-for-chrome' do
+  version '1_5_0_9'
+  sha256 '01773a09ae8d049bbd70a2d9753f16f5795b77621ad61b46f09e0e08b99207a0'
+
+  url "http://portalapp.dmm.com/silverlightplayer/dmm/m/#{version}/DMMPlayerForChromeInstaller_#{version}.pkg"
+  name 'DMM Player for Chrome'
+  homepage 'http://www.dmm.co.jp/digital/'
```
Comment (vitorgalvao) [inline Casks/dmm-player-for-chrome.rb]: This page does not work for me.

File: Casks/dmm-player-for-chrome.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'dmm-player-for-chrome' do
+  version '1_5_0_9'
```
Comment (vitorgalvao) [inline Casks/dmm-player-for-chrome.rb]: [Same logic](https://github.com/caskroom/homebrew-cask/pull/20928/files#r61792487).

**Comment by yutarody** [comment]
Thank you for comment & check.
Sorry I was careless.
I was fixed because there were some mistakes

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**